spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -2563,7 +2563,8 @@ def setup_firefox_with_linux
   describe "#build_toolbar_hide_button_ops" do
     subject { build_toolbar_hide_button_ops(@id) }
     before do
-      @record = FactoryGirl.create(:tenant)
+      Tenant.seed
+      @record = FactoryGirl.create(:tenant, :parent => Tenant.root_tenant)
```

Comment:
this is good, or you may want to condense both into a single line? (if you don't that is good too)
```ruby
@record = FactoryGirl.create(:tenant, :parent => Tenant.seed)
```